### DISCUSSION TITLE
自己构建镜像 apt-get update 总是失败

### DISCUSSION THREAD

**Discussion by liuxinwang** [hhyo/Archery#2683] [General]
因为mongodb无法运行，看issue里面需要自己构建镜像

`docker build -f Dockerfile-base -t hhyo/archery-base:sha-d8159f4 .`

37.81 + apt-get update
38.05 Err:1 http://deb.debian.org/debian bullseye InRelease
38.05   500  writing response to deb.debian.org:80: reading HTTP GET response: unexpected EOF [IP: 151.101.110.132 80]
38.22 Err:2 http://deb.debian.org/debian-security bullseye-security InRelease
38.22   500  writing response to deb.debian.org:80: reading HTTP GET response: unexpected EOF [IP: 151.101.110.132 80]
38.42 Get:3 https://packages.microsoft.com/debian/11/prod bullseye InRelease [3650 B]
38.44 Err:4 http://deb.debian.org/debian bullseye-updates InRelease
38.44   500  writing response to deb.debian.org:80: reading HTTP GET response: unexpected EOF [IP: 151.101.110.132 80]
......
49.83 Reading package lists...
49.85 Building dependency tree...
49.85 Reading state information...
49.86 E: Unable to locate package libaio1

**Comment by LeoQuote** [comment]
加代理或者换镜像，自己搜一下
